docs/reverse.rst

```diff
@@ -221,6 +221,68 @@ Alternatively, you can define SANs in the Traefik static configuration using rou
 
 .. _`Traefik`: https://traefik.io/
 
+Linuxserver Swag reverse proxy
+------------------------------
+
+Another popular reverse proxy is Linuxserver Swag with built-in ``letsencrypt``. It can be placed in front of the Mailu front container and use with other docker-containers in ``docker-compose.yml``:
```

Comment:
It can be placed in front of the Mailu front container and `used` with other docker-containers in ....

Reply:
> I'm not sure if we should document this. If you really need this much help to set up a reverse proxy (besides the existing documentation), then you are not the intended audience. You should not use this if you have no clue what you are doing.

Sorry, I don't get it. Traefik is also documented, so why can't Linuxserver Swag not documented? The Swag configuration is much easier.

> It can be placed in front of the Mailu front container and used with other docker-containers in ....

* [x] Fixed typo

> Double check in the swag config what header is used. 

Same `proxy_set_header X-Real-IP $remote_addr;`: 
https://github.com/linuxserver/docker-swag/blob/master/root/defaults/proxy.conf

* [x] Then enhance the docs how to configure these two settings in mailu.env.

> Without these settings Front will not trust the remote client ip that swag forwards to front. This is a new feature in master/1.9.

Ok, I missed this in the new version.

> I must still test the PR and read it via the docs container.

A screenshot part:

![image](https://user-images.githubusercontent.com/8849873/147272906-c6b7ddd0-7244-496e-ad66-84527772e373.png)

...

Reply:
It keeps working with:

```bash
REAL_IP_HEADER=X-Real-IP
REAL_IP_FROM=x.x.x.x,y.y.y.y.y
```

However, I have to configure a fixed IP address for Swag: